15-PageBehaviorFactory/src/PrismSample/App.xaml.cs

```diff
@@ -1,6 +1,6 @@
-﻿using System;
+﻿using Prism.Behaviors;
 using Prism.Ioc;
-using PrismSample.ViewModels;
+using PrismSample.Behaviours;
```

Comment:
I realize that Behaviours is technically correct, but let's keep the naming consistent with Prism and use Behaviors